chroma_core/models/filesystem.py

```diff
@@ -97,6 +97,9 @@ def get_servers(self):
         # (http://stackoverflow.com/questions/4764110/django-template-cant-loop-defaultdict)
         return dict(servers)
 
+    def get_pools(self):
+        return OstPool.objects.filter(Q(filesystem=self))
```

Comment:
Is the `Q` necessary? There are many places where we filter without the Q.

Reply:
I'm not sure.  I just copied a filter line from somewhere else, I'll remove the Q()